cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -1302,7 +1302,9 @@ data:
                 "carbonEstimatesEnabled": "{{ template "carbonEstimatesEnabled" . }}",
                 "clusterControllerEnabled": "{{ template "clusterControllerEnabled" . }}",
                 "forecastingEnabled": "{{ template "forecastingEnabled" . }}",
-                "chartVersion": "DEVELOP_BRANCH"
+                "chartVersion": "DEVELOP_BRANCH",
+                "hourlyDataRetention": "{{ (.Values.kubecostModel.etlHourlyStoreDurationHours) }}",
+                "dailyDataRetention": "{{ (.Values.kubecostModel.etlDailyStoreDurationDays) }}"
```

Comment:
My only concern here, is that this `.Values.kubecostModel.etlDailyStoreDurationDays` is now an overloaded variable (as mentioned in PR https://github.com/kubecost/cost-analyzer-helm-chart/pull/3399). And making a change like this just digs us deeper into how overloaded the variable has become. (similar to how `.Values.prometheus.server.global.external_labels` is overloaded).

Technically when this env var is set in Aggregator, it does indeed define what the retention is of the database. In cost-model and cloudcost it determines how far back to build/push ETL.

I'm ok approving for now since this endpoint is only used by FE. But I think we should be mindful of changes like this.